<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QGroupBox" name="groupBox_view">
      <property name="title">
       <string>Affichage</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_viewStyle">
         <property name="text">
          <string>Style</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_viewStyle"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="checkBox_keepViewSize">
         <property name="text">
          <string>Conserver la taille de la fenêtre entre deux lancements</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QGroupBox" name="groupBox_compiler">
      <property name="title">
       <string>Compilateur</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_headersExt"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_headersExt">
         <property name="text">
          <string>Extensions des headers</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_sourcesExt">
         <property name="text">
          <string>Extensions des sources</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_sourcesExt"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButton_apply">
      <property name="text">
       <string>Appliquer</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="text">
       <string>Fermer</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QGroupBox" name="groupBox_doxygen">
      <property name="title">
       <string>Doxygen</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_doxygenTemplate"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_doxygenTemplate">
         <property name="text">
          <string>Dossier des template</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Displayers</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_colorNormal">
         <property name="text">
          <string>Couleur par défaut</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_colorNormal"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_colorSuccess">
         <property name="text">
          <string>Couleur de succès</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_colorSuccess"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_colorError">
         <property name="text">
          <string>Couleur d'erreur</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBox_colorError"/>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="checkBox_autoclearScreen">
         <property name="text">
          <string>Effacer l'écran entre deux actions</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
